codespell_lib/data/dictionary.txt

```diff
@@ -2054,6 +2057,7 @@ basf->base
 basicaly->basically
 basicly->basically
 batery->battery
+batter->better, battery,
```

Comment:
Actually sorry, remove this please, batter is a valid word ( https://en.wikipedia.org/wiki/Batter_(cooking) ).

Reply:
No prob.

Removed.